CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 We follow [Semantic Versioning](http://semver.org/) as a way of measuring stability of an update. This
 means we will never make a backwards-incompatible change within a major version of the project.
 
+## Unreleased (2018-1-15)
```

Comment:
If we're gonna have it be in an "unreleased" section, we should avoid the datestamping here. Think about it this way:

> "When was this 'Unreleased'? Oh, on that day? What about after that date: was it released then?"

The datestamping on the ones listed below with version numbers are just a convenience thing. They are a snapshot in time and we're noting when the snapshot was taken along with the semantic versioning style of version number.